### PR TITLE
This PR changes the path counter for a timestamp. The counter was subscribing another paths.

### PR DESCRIPTION
**Pull request by gabrielsscavalcante**
### Platforms affected

iOS

### What does this PR do?

This PR changes the path counter for a timestamp. The counter was subscribing another paths.

### What testing has been done on this change?

Only used on my application.

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-camera",
-  "version": "4.0.4-dev",
+  "version": "~4.0.2",
```
Comment (janpio) [inline package.json]: This breaks all the tests as it is not a valid version.

File: package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
       },
       "4.1.0": {
         "cordova-android": ">=6.3.0",
-        "cordova": ">=7.1.0"
+        "cordova": ">=6.5.0"
```
Comment (janpio) [inline package.json]: Why this change?

**Comment by jcesarmobile** [comment]
this PR did a similar thing https://github.com/apache/cordova-plugin-camera/pull/306 and it's merged, closing
